src/java.desktop/windows/classes/sun/java2d/opengl/WGLGraphicsConfig.java

```diff
@@ -432,4 +433,9 @@ public ImageCapabilities getImageCapabilities() {
     public ContextCapabilities getContextCapabilities() {
         return oglCaps;
     }
+
+    @Override
+    public VolatileSurfaceManager createVolatileManager(SunVolatileImage image, Object context) {
+        return new WGLVolatileSurfaceManager(image, context);
+    }
```

Comment:
Please split long lines to have 80 chars per line, it will be good to split surfaceDataProxyCache line added by the 8339341 as well.